code/modules/mining/abandoned_crates.dm

```diff
@@ -149,6 +149,293 @@
+			new /obj/item/clothing/accessory/medal/silver/excellence(src)
+			new /obj/item/clothing/accessory/medal/silver/security(src)
+			new /obj/item/clothing/accessory/medal/silver/valor(src)
+		if(112)
+			new /obj/item/clothing/accessory/medal/gold/heroism(src)
+			new /obj/item/clothing/accessory/medal/gold/captain(src)
+			new /obj/item/reagent_containers/food/drinks/trophy/gold_cup(src)
+			new /obj/item/reagent_containers/food/snacks/grown/apple/gold(src)
+		if(113)//Lite, mid and heavy are Snowndin lootdrops. Putting a spawner into the chest result things spawn on top of it instead of inside.
+			switch(pickweight(list("heavystuff1" = 4, "heavystuff2" = 3, "heavystuff3" = 1, "heavystuff4" = 4, "heavystuff5" = 2, "heavystuff6" = 1, "heavystuff7" = 2, "heavystuff8" = 1)))
+				if("heavystuff1")
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+					new /obj/item/banhammer(src)
+					new /obj/item/clothing/head/helmet/riot(src)
+				if("heavystuff2")
+					new /obj/item/clothing/head/helmet/roman/legionnaire(src)
+					new /obj/item/clothing/shoes/roman(src)
+					new /obj/item/clothing/under/costume/roman(src)
+					new /obj/item/shield/riot/roman(src)
+					new /obj/item/melee/shank(src)
+				if("heavystuff3")
+					new /obj/item/twohanded/fireaxe(src)
+					new /obj/item/organ/brain/alien(src)
+				if("heavystuff4")
+					new /obj/item/modular_computer/tablet/preset/advanced(src)
+				if("heavystuff5")
+					new /obj/item/stack/spacecash/c1000(src)
+					new /obj/item/stack/spacecash/c1000(src)
+					new /obj/item/clothing/head/collectable/tophat(src)
+					new /obj/item/storage/fancy/cigarettes/cigars/cohiba(src)
+				if("heavystuff6")
+					new /obj/item/borg/upgrade/syndicate(src)
+				if("heavystuff7")
+					new /obj/item/borg/upgrade/selfrepair(src)
+				if("heavystuff8")
+					new /obj/item/clothing/suit/armor/riot/knight/red(src)
+					new /obj/item/clothing/gloves/plate/red(src)
+					new /obj/item/clothing/shoes/plate/red(src)
+					new /obj/item/clothing/head/helmet/knight/red(src)
+					new /obj/item/kitchen/knife/butcher(src)
+		if(114)
+			for(var/food in 5)
+				food = pick(subtypesof(/obj/item/reagent_containers/food/snacks) - /obj/item/reagent_containers/food/snacks)
+				new food(src)
+		if(115)
+			new /obj/item/kitchen/knife/rainbowknife(src)//Yooo my 400000$ knife lets sell it and become rich!!!!11
+		if(116)
+			switch(pickweight(list("litestuff" = 3, "litestuff2" = 2, "litestuff3" = 1, "litestuff4" = 4, "litestuff5" = 1)))
+				if("litestuff")
+					new /obj/item/melee/classic_baton/police(src)
+					new /obj/item/storage/firstaid/regular(src)
+					new /obj/item/storage/toolbox/syndicate(src)
```

Comment:
Antagonist items should not be in these crates considering they are easy to open and encourage meta programs